src/lib/y2s390/dasd_actions/format.rb

```diff
@@ -0,0 +1,84 @@
+require "y2s390/dasd_actions/base"
+
+module Y2S390
+  module DasdActions
+    class FormatOff < Base
```

Comment:
I guess you actually wanted to inherit from Format here.

Reply:
:thinking: Or maybe not, since this action performs a different #run. Sorry, I got confused with the "analogous" DiagOff.